app/src/main/java/com/kickstarter/viewmodels/LoggedInViewHolderViewModel.kt

```diff
@@ -0,0 +1,97 @@
+package com.kickstarter.viewmodels
+
+import androidx.annotation.NonNull
+import com.kickstarter.libs.ActivityViewModel
+import com.kickstarter.libs.Environment
+import com.kickstarter.libs.utils.IntegerUtils
+import com.kickstarter.models.User
+import com.kickstarter.ui.viewholders.discoverydrawer.LoggedInViewHolder
+import rx.Observable
+import rx.subjects.BehaviorSubject
+import rx.subjects.PublishSubject
+
+interface LoggedInViewHolderViewModel {
+
+    interface Inputs {
+        /** Call with the current user when the data is bound to the view. */
+        fun configureWith(user: User)
+    }
+
+    interface Outputs {
+        /** Emits the user's medium avatar URL. */
+        fun avatarUrl(): Observable<String>
+
+        /** Emits a boolean determining if the user is a creator. */
```

Comment:
Here I would say that the Observable emits a boolean determining if the dashboardRow should be gone, depending if the user is creator or not.